warehouse/core/pom.xml

```diff
@@ -55,6 +55,16 @@
         <dependency>
             <groupId>gov.nsa.datawave.microservice</groupId>
             <artifactId>accumulo-utils</artifactId>
+            <exclusions>
```

Comment:
I wonder if the accumulo-utils pom marking certain deps as scope provided would allow dropping these exclusions.  For example if accumulo-utils marked zookeeper as provided in its pom then maybe zookeeper and zookeepers deps would not show up as a transitive dep for everything that uses accumulo-utils.  Not something that has to be done for this PR.

Reply:
Zookeeper is listed as a dependency in the parent datawave pom so I think that would work.
I'd rather stop the transitive dep from being included initially vs adding all of these exclusions.